spec/services/unwithdraw_service_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+RSpec.describe UnwithdrawService do
+  before do
+    @user = create(:user)
+  end
+
+  describe "#call" do
```

Comment:
The timeline and status stuff is lovely, but are we missing a test for the actual call to republish?

Reply:
I've added one in